deploy.sh

```diff
@@ -18,6 +19,11 @@ RED='\033[1;31m'
 PUR="\033[1;35m"
 NRM='\033[0m'
 
+trap ctrl_c INT
+function ctrl_c() {
+  echo "User interrupted termination!"
```

Comment:
Do we want to show a warning and the confirm again to really cancel it?

Reply:
can be done, will check